### PR TITLE
Added a wiki to outline the triage process.

### PR DESCRIPTION
**Pull request by Lawful2002**
This PR adds a new wiki to outline the steps to triage issues according to [this doc](https://docs.google.com/document/d/1MuNXlygGNaeLYtByw1eZE6YCkLh_rccBXcQvKOw-PhM/edit#heading=h.lkpm971gy70c).

### CODE REVIEW

**Comment by Lawful2002** [comment]
@seanlip PTAL.

File: Triaging-process.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- Assign the issue to the appropriate task force, if applicable. Otherwise, assign it to the relevant team (LaCE, CD, Dev Workflow) based on which type of user it affects.
+- If you’re not sure how to categorise an issue, feel free to ping Sean for help!
+
+### Steps to triage incoming issues for each team (second-stage triage):
+Team leads should do an audit every week to categorise new issues in their GitHub Projects board, following these steps:
+
+- Verify that the issue lies within the scope of the team (see definitions below).
+    - If not, assign the issue to the appropriate project/team.
+- Determine whether the issue is a bug or a enhancement
+    - If the issue is a feature request (enhancement), contact the relevant PM (product manager) for the team and ask them to take a look and decide whether it needs to be worked upon. (See the list of PMs [below].)
```
Comment (seanlip) [inline Triaging-process.md]: Can you make "below" a link?

File: Triaging-process.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+- (For simple UI issues) Make sure the bug is reproducible
+    - The issue should be consistently reproducible on either develop or the test/backup server.
+    - If that isn’t the case ask the issue author for clarity
```
Comment (seanlip) [inline Triaging-process.md]: If that isn't the case, ask the issue author for clarity.

(missing comma and period)

File: Triaging-process.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            <td>Medium</td>
+            <td>Low</td>
+            <td>Low</td>
+        </tr>
+    </table>
+
+    <p style="text-align:center; font-size:12px"> Table: How “impact” is determined </p>
+
+    - ‘Work’ defines how hard it would be to solve/resolve a particular issue. This is generally correlated with the amount of ambiguity in the task. We use the following rough system for assigning this label:
+        - Solution is clear and broken into good-first-issue-sized chunks – assign Work: Low
```
Comment (seanlip) [inline Triaging-process.md]: Here and elsewhere, make label names (e.g. "Work: Low") bold

File: Triaging-process.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        - Otherwise, if the means to find the solution is clear – assign Work: Medium
+        - Everything else – assign Work: High
+
+- Once the issue has been confirmed and the appropriate ‘Impact’ and ‘Work Required’ has been added, determine its severity and assign the fitting priority label based on its impact (and possibly the work required label)
```
Comment (seanlip) [inline Triaging-process.md]: ...'Work Required' labels have been added, ...

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks @Lawful2002! Left a few notes :)

File: Triaging-process.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        <tr>
+            <td>(roughly) &lt; 5%</td>
+            <td></td>
+            <td>Medium</td>
+            <td>Low</td>
+            <td>Low</td>
+        </tr>
+    </table>
+
+    <p style="text-align:center; font-size:12px"> Table: How “impact” is determined </p>
```
Comment (gp201) [inline Triaging-process.md]: Can you try using the [markdown table syntax](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables)?

File: Triaging-process.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+The Dev Workflow team lead (currently Chris) should do the following every week for any [filed issues that aren’t assigned to a project](https://github.com/oppia/oppia/issues?q=is%3Aissue+is%3Aopen+no%3Aproject):
+
+- Check that the issue is written clearly enough. Request clarification if needed.
+- Assign the issue to the appropriate task force, if applicable. Otherwise, assign it to the relevant team (LaCE, CD, Dev Workflow) based on which type of user it affects.
+- If you’re not sure how to categorise an issue, feel free to ping Sean for help!
+
+### Steps to triage incoming issues for each team (second-stage triage):
+Team leads should do an audit every week to categorise new issues in their GitHub Projects board, following these steps:
+
+- Verify that the issue lies within the scope of the team (see definitions below).
```
Comment (gp201) [inline Triaging-process.md]: Please number the steps

File: Triaging-process.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+        - For such issues, also assign purple “technology: TypeScript”, “technology: Apache Beam”, “technology: Python”, etc. labels as appropriate. This helps new contributors identify issues that they would find approachable.
+
+After triaging all issues, make sure the issues in the Priority and High Priority Buckets are sorted based on severity/priority (high to low).
+
+
+## PM contacts for each team
+- **LaCE Quality**: Diana (learners), Sean (creators)
+- **Contributor dashboard**: Diana
+- **Dev workflow**: Chris
+- **Android**: Sean
```
Comment (gp201) [inline Triaging-process.md]: Try mentioning the GitHub id, so it will be easier for users to identify the GitHub ID.

File: Triaging-process.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+    - Other issues go in the ‘backlog’ bucket.
+    - Any issue with an external dependency that the team cannot resolve by itself (eg. missing mocks) goes in the ‘blocked’ bucket, and the reason should be specified on the ‘Issue specific status’ field (using free-text). Team leads should make sure an issue spends as little time here as possible.
+
+- Determine if the issue is potentially a good first issue
+    - If the issue seems like something a new contributor to Oppia can take up, make sure to add enough information in the issue description for a new contributor to start work on that issue then ask Sandeep or Vojta to add the Good First Issue label. (Note: typically these are “Work: Low” issues.)
+        - For such issues, also assign purple “technology: TypeScript”, “technology: Apache Beam”, “technology: Python”, etc. labels as appropriate. This helps new contributors identify issues that they would find approachable.
+
+After triaging all issues, make sure the issues in the Priority and High Priority Buckets are sorted based on severity/priority (high to low).
```
Comment (gp201) [inline Triaging-process.md]: Is this a step or a note?

**Review by gp201 — CHANGES_REQUESTED** [review]
Thanks, @Lawful2002 ! I left some comments. PTAL

File: Triaging-process.md (hunk 1 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

File: Triaging-process.md (hunk 2 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

File: Triaging-process.md (hunk 3 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

File: Triaging-process.md (hunk 4 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

File: Triaging-process.md (hunk 5 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

File: Triaging-process.md (hunk 6 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

File: Triaging-process.md (hunk 7 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

File: Triaging-process.md (hunk 8 above)
Comment (Lawful2002) [reply Triaging-process.md]: It is a step, made the necessary changes.

**Comment by Lawful2002** [comment]
@seanlip @gp201 I have addressed the changes PTAL.

File: Triaging-process.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+    - If the issue is a feature request (enhancement), contact the relevant PM (product manager) for the team and ask them to take a look and decide whether it needs to be worked upon. (See the list of PMs [below](##pm-contacts-for-each-team).)
+3. (For simple UI issues) Make sure the bug is reproducible
+    - The issue should be consistently reproducible on either develop or the test/backup server.
+    - If that isn’t the case, ask the issue author for clarity.
+4. Once the issue has been confirmed, the team lead should add the  appropriate ‘Impact’ label (asking the PM or tech lead for help if they’re not sure), as well as the appropriate ‘Work’ label to the issue.
+    - ‘Impact’ defines how much the issue affects the functionality of the user-facing features. There are 3 associated labels: **Impact: High**, **Impact: Medium** and **Impact: Low**. For eg., the issue is considered to have a high impact if it breaks a critical feature or blocks a release. General bugs (non-blocking) and improvements can be considered to be of medium impact while issues that occur in a few edge cases or affect a small number of users are issues with a low impact.
```
Comment (seanlip) [inline Triaging-process.md]: For eg. --> For example

**Review by seanlip — APPROVED** [review]
@Lawful2002 Just one minor comment, otherwise LGTM. Thanks!

**Review by seanlip — CHANGES_REQUESTED** [review]
Actually @Lawful2002 one sec. Don't you need to add it to _Sidebar.md as well?

File: Triaging-process.md (hunk 9 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

**Comment by Lawful2002** [comment]
@seanlip I have added the file to the sidebar as well. PTAL.

**Review by seanlip — APPROVED** [review]
Thanks @Lawful2002! LGTM from my end :)

File: Triaging-process.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+        - For such issues, also assign purple “technology: TypeScript”, “technology: Apache Beam”, “technology: Python”, etc. labels as appropriate. This helps new contributors identify issues that they would find approachable.
+
+7. After triaging all issues, make sure the issues in the 'Priority' and 'High Priority' buckets are sorted based on severity/priority (high to low).
+
+
+## PM contacts for each team
+- **LaCE Quality**: @dchen97 (learners), @seanlip (creators)
+- **Contributor dashboard**: @dchen97
+- **Dev workflow**: @U8NWXD
+- **Android**: @seanlip
```
Comment (gp201) [inline Triaging-process.md]: Sorry for not being clear. Can you mention it in this format: "<Name> (GitHub_ID)"

File: Triaging-process.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+    - Other issues go in the ‘backlog’ bucket.
+    - Any issue with an external dependency that the team cannot resolve by itself (eg. missing mocks) goes in the ‘blocked’ bucket, and the reason should be specified on the ‘Issue specific status’ field (using free-text). Team leads should make sure an issue spends as little time here as possible.
+
+6. Determine if the issue is potentially a good first issue
+    - If the issue seems like something a new contributor to Oppia can take up, make sure to add enough information in the issue description for a new contributor to start work on that issue then ask Sandeep or Vojta to add the Good First Issue label. (Note: typically these are “Work: Low” issues.)
+        - For such issues, also assign purple “technology: TypeScript”, “technology: Apache Beam”, “technology: Python”, etc. labels as appropriate. This helps new contributors identify issues that they would find approachable.
```
Comment (gp201) [inline Triaging-process.md]: What type of issues? (Good first issues?)

File: Triaging-process.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+    - ‘Impact’ defines how much the issue affects the functionality of the user-facing features. There are 3 associated labels: **Impact: High**, **Impact: Medium** and **Impact: Low**. For example, the issue is considered to have a high impact if it breaks a critical feature or blocks a release. General bugs (non-blocking) and improvements can be considered to be of medium impact while issues that occur in a few edge cases or affect a small number of users are issues with a low impact.
+
+    ||||||
+    |--- |--- |--- |--- |--- |
+    ||Severity|Severe (blocks or represents a primary user journey – e.g. end user playing a lesson, developer installing Oppia)|Moderate severity (blocks or represents a secondary user journey – e.g. end user changing a profile picture, developer easily seeing the output of frontend tests at the bottom of the CI log)|Minor (neither of the others, relates to “polish”)|
+    |Users Affected|||||
+    |(roughly) 60+%||High|Medium|Low|
+    |(roughly) 5-60%||High|Low|Low|
+    |(roughly) < 5%||Medium|Low|Low|
```
Comment (gp201) [inline Triaging-process.md]: Please fix the table.

File: Triaging-process.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+        - Otherwise, if the means to find the solution is clear – assign **Work: Medium**
+        - Everything else – assign **Work: High**
+
+5. Once the appropriate ‘Impact’ and ‘Work Required’ labels has been added, determine its severity and assign the fitting priority label based on its impact (and possibly the work required label)
+    - If the issue is a high-impact bug, put it in the high-priority bucket and prioritise it within the team.
+    - For the priority bucket, team leads should include “important” issues – typically, a selection of high-impact enhancements and low-work medium-impact bugs – while keeping the size of that bucket about 2-3 times the projected stable number of team members. (Note: This exercise might also help with determining how large the team itself needs to be.)
+    - Other issues go in the ‘backlog’ bucket.
+    - Any issue with an external dependency that the team cannot resolve by itself (eg. missing mocks) goes in the ‘blocked’ bucket, and the reason should be specified on the ‘Issue specific status’ field (using free-text). Team leads should make sure an issue spends as little time here as possible.
```
Comment (gp201) [inline Triaging-process.md]:
```suggestion
    - Any issue with an external dependency that the team cannot resolve by itself (example: missing mocks) goes in the ‘blocked’ bucket, and the reason should be specified in the ‘Issue specific status’ field (using free-text). Team leads should ensure an issue spends as little time here as possible.
```

File: Triaging-process.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+
+    - ‘Work’ defines how hard it would be to solve/resolve a particular issue. This is generally correlated with the amount of ambiguity in the task. We use the following rough system for assigning this label:
+        - Solution is clear and broken into good-first-issue-sized chunks – assign **Work: Low**
+        - Otherwise, if the means to find the solution is clear – assign **Work: Medium**
+        - Everything else – assign **Work: High**
+
+5. Once the appropriate ‘Impact’ and ‘Work Required’ labels has been added, determine its severity and assign the fitting priority label based on its impact (and possibly the work required label)
+    - If the issue is a high-impact bug, put it in the high-priority bucket and prioritise it within the team.
+    - For the priority bucket, team leads should include “important” issues – typically, a selection of high-impact enhancements and low-work medium-impact bugs – while keeping the size of that bucket about 2-3 times the projected stable number of team members. (Note: This exercise might also help with determining how large the team itself needs to be.)
```
Comment (gp201) [inline Triaging-process.md]:
```suggestion
    - For the priority bucket, team leads should include “important” issues – typically, a selection of high-impact enhancements and low-work medium-impact bugs – while keeping the size of that bucket about 2-3 times the projected stable number of team members.
    > **Note**
    > This exercise might also help with determining how large the team itself needs to be.
```

File: Triaging-process.md [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+
+    ||||||
+    |--- |--- |--- |--- |--- |
+    ||Severity|Severe (blocks or represents a primary user journey – e.g. end user playing a lesson, developer installing Oppia)|Moderate severity (blocks or represents a secondary user journey – e.g. end user changing a profile picture, developer easily seeing the output of frontend tests at the bottom of the CI log)|Minor (neither of the others, relates to “polish”)|
+    |Users Affected|||||
+    |(roughly) 60+%||High|Medium|Low|
+    |(roughly) 5-60%||High|Low|Low|
+    |(roughly) < 5%||Medium|Low|Low|
+
+    <p style="text-align:center; font-size:12px"> Table: How “impact” is determined </p>
```
Comment (gp201) [inline Triaging-process.md]: Do we need this to be in the center? If so, please fix it.

File: Triaging-process.md [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+- Assign the issue to the appropriate task force, if applicable. Otherwise, assign it to the relevant team (LaCE, CD, Dev Workflow) based on which type of user it affects.
+- If you’re not sure how to categorise an issue, feel free to ping Sean for help!
+
+### Steps to triage incoming issues for each team (second-stage triage):
+Team leads should do an audit every week to categorise new issues in their GitHub Projects board, following these steps:
+
+1. Verify that the issue lies within the scope of the team (see definitions below).
+    - If not, assign the issue to the appropriate project/team.
+2. Determine whether the issue is a bug or a enhancement
+    - If the issue is a feature request (enhancement), contact the relevant PM (product manager) for the team and ask them to take a look and decide whether it needs to be worked upon. (See the list of PMs [below](##pm-contacts-for-each-team).)
```
Comment (gp201) [inline Triaging-process.md]: I don't think the `[below](##pm-contacts-for-each-team)` link is working please fix it.

**Review by gp201 — CHANGES_REQUESTED** [review]
Left a few comments.
FYI, you can see how the page renders in vscode by clicking Ctrl + shift + K. ([Source](https://code.visualstudio.com/docs/languages/markdown#_markdown-preview))
Or in GitHub by going to your [repo](https://github.com/Lawful2002/oppia-web-developer-docs/blob/triage/Triaging-process.md)

File: Triaging-process.md (hunk 11 above)
Comment (Lawful2002) [reply Triaging-process.md]: Yes these refer to good first issues. Should I explicitly mention that, as they are already nested under a related point?

File: Triaging-process.md (hunk 15 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

File: Triaging-process.md (hunk 16 above)
Comment (Lawful2002) [reply Triaging-process.md]: Fixed

File: Triaging-process.md (hunk 10 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

File: Triaging-process.md (hunk 12 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

File: Triaging-process.md [hunk 17]
```diff
@@ -0,0 +1,63 @@
...
+
+        || Severity  | Severe (blocks or represents a primary user journey – e.g. end user playing a lesson, developer installing Oppia) | Moderate severity (blocks or represents a secondary user journey – e.g. end user changing a profile picture, developer easily seeing the output of frontend tests at the bottom of the CI log) | Moderate severity (blocks or represents a secondary user journey – e.g. end user changing a profile picture, developer easily seeing the output of frontend tests at the bottom of the CI log) |
+        | -------------------  | ------------------- | --------------------- | ------------------| ---------- |
+        | **# users affected**|        |        |     |
+        | (roughly) 60+%      || High   | Medium | Low |
+        | (roughly) 5-60%     || High   | Low    | Low |
+        | (roughly) < 5%      || Medium | Low    | Low |
```
Comment (gp201) [inline Triaging-process.md]: This seems to be a lot of information to take in, especially the headers. Can we simplify the format?

File: Triaging-process.md (hunk 11 above)
Comment (gp201) [reply Triaging-process.md]: Just to rule out any possibility of confusion. change `For such issues` to `For such good first issues`

File: Triaging-process.md (hunk 17 above)
Comment (gp201) [reply Triaging-process.md]: Also, the `Moderate severity` seems to be repeated twice. is this intentional?

**Review by gp201 — CHANGES_REQUESTED** [review]
A couple of more changes

File: Triaging-process.md (hunk 17 above)
Comment (Lawful2002) [reply Triaging-process.md]: Fixed.

**Comment by Lawful2002** [comment]
@gp201 I have made some changes to the table layout. PTAL.

File: Triaging-process.md [hunk 18]
```diff
@@ -0,0 +1,73 @@
...
+4. Once the issue has been confirmed, the team lead should add the  appropriate ‘Impact’ label (asking the PM or tech lead for help if they’re not sure), as well as the appropriate ‘Work’ label to the issue.
+    - ‘Impact’ defines how much the issue affects the functionality of the user-facing features. There are 3 associated labels: **Impact: High**, **Impact: Medium** and **Impact: Low**. For example, the issue is considered to have a high impact if it breaks a critical feature or blocks a release. General bugs (non-blocking) and improvements can be considered to be of medium impact while issues that occur in a few edge cases or affect a small number of users are issues with a low impact.
+
+        - Severity:
+            - **Severe**: Blocks or represents a primary user journey – e.g. end user playing a lesson, developer installing Oppia.
+            - **Moderate**: blocks or represents a secondary user journey – e.g. end user changing a profile picture, developer easily seeing the output of frontend tests at the bottom of the CI log.
+            - **Minor**: Neither of the others, relates to “polish”.
+
+        <br />
```
Comment (gp201) [inline Triaging-process.md]: Do we need this?

File: Triaging-process.md [hunk 19]
```diff
@@ -0,0 +1,73 @@
...
+        |     | Severity       | Severe | Moderate | Minor |
+        | --- | -------------- | ------ | -------- | ----- |
+        | **# users affected** |        |          |       |
+        | (roughly) 60+%      || High   | Medium   | Low   |
+        | (roughly) 5-60%     || High   | Low      | Low   |
+        | (roughly) < 5%      || Medium | Low      | Low   |
+
+        Table: How “impact” is determined.
+
+        <br />
```
Comment (gp201) [inline Triaging-process.md]: Do we need this?

File: Triaging-process.md [hunk 20]
```diff
@@ -0,0 +1,73 @@
...
+
+    - ‘Work’ defines how hard it would be to solve/resolve a particular issue. This is generally correlated with the amount of ambiguity in the task. We use the following rough system for assigning this label:
+        - Solution is clear and broken into good-first-issue-sized chunks – assign **Work: Low**
+        - Otherwise, if the means to find the solution is clear – assign **Work: Medium**
+        - Everything else – assign **Work: High**
```
Comment (gp201) [inline Triaging-process.md]: Mention the `Work` label at the beginning of the point and then explain when to assign it.

File: Triaging-process.md [hunk 21]
```diff
@@ -0,0 +1,73 @@
+## General
+This wiki outlines the steps to triage new and existing issues. This ensures that all issues are being effectively tracked and prioritized.
+
+### Steps to triage incoming issues coming into Oppia as a whole (first-stage triage):
```
Comment (gp201) [inline Triaging-process.md]: Probably mention "first-stage triage" at the beginning. So the reader knows that there are 2 stages.

File: Triaging-process.md [hunk 22]
```diff
@@ -0,0 +1,73 @@
...
+The Dev Workflow team lead (currently Chris) should do the following every week for any [filed issues that aren’t assigned to a project](https://github.com/oppia/oppia/issues?q=is%3Aissue+is%3Aopen+no%3Aproject):
+
+- Check that the issue is written clearly enough. Request clarification if needed.
+- Assign the issue to the appropriate task force, if applicable. Otherwise, assign it to the relevant team (LaCE, CD, Dev Workflow) based on which type of user it affects.
+- If you’re not sure how to categorise an issue, feel free to ping Sean (@seanlip) for help!
+
+### Steps to triage incoming issues for each team (second-stage triage):
```
Comment (gp201) [inline Triaging-process.md]: *Ditto

File: Triaging-process.md [hunk 23]
```diff
@@ -0,0 +1,73 @@
...
+        - Otherwise, if the means to find the solution is clear – assign **Work: Medium**
+        - Everything else – assign **Work: High**
+
+5. Once the appropriate ‘Impact’ and ‘Work Required’ labels has been added, determine its severity and assign the fitting priority label based on its impact (and possibly the work required label)
+    - If the issue is a high-impact bug, put it in the high-priority bucket and prioritise it within the team.
+    - For the priority bucket, team leads should include “important” issues – typically, a selection of high-impact enhancements and low-work medium-impact bugs – while keeping the size of that bucket about 2-3 times the projected stable number of team members.
+    > **Note**
+    > This exercise might also help with determining how large the team itself needs to be.
+    - Other issues go in the ‘backlog’ bucket.
+    - Any issue with an external dependency that the team cannot resolve by itself (example: missing mocks) goes in the ‘blocked’ bucket, and the reason should be specified in the ‘Issue specific status’ field (using free-text). Team leads should ensure an issue spends as little time here as possible.
```
Comment (gp201) [inline Triaging-process.md]:
```suggestion
      > **Note**
      > This exercise might also help determine how large the team needs to be.

    - Other issues go in the ‘backlog’ bucket.
    - Any issue with an external dependency that the team cannot resolve by itself (example: missing mocks) goes in the ‘blocked’ bucket, and the reason should be specified in the ‘Issue specific status’ field (using free-text). Team leads should ensure an issue spends as little time here as possible.
```

**Review by gp201 — CHANGES_REQUESTED** [review]
Thanks! I left a few more comments

File: Triaging-process.md (hunk 18 above)
Comment (Lawful2002) [reply Triaging-process.md]: I'm not really sure, but I think this provides the reader about the 'scope' of multiple levels of severity.

File: Triaging-process.md (hunk 19 above)
Comment (Lawful2002) [reply Triaging-process.md]: I think we would need some way to tell the readers the use of this table.

File: Triaging-process.md (hunk 18 above)
Comment (gp201) [reply Triaging-process.md]: Sorry for not being clear. I meant the ` <br />` tag.

File: Triaging-process.md (hunk 19 above)
Comment (gp201) [reply Triaging-process.md]: Sorry for not being clear. I meant the ` <br />` tag.

**Comment by gp201** [comment]
@Lawful2002 replied to your comments.

File: Triaging-process.md (hunk 18 above)
Comment (Lawful2002) [reply Triaging-process.md]: I have replaced the table with an image. The `<br />` tag is no longer needed.

File: Triaging-process.md (hunk 19 above)
Comment (Lawful2002) [reply Triaging-process.md]: I have replaced the table with an image. The `<br />` tag is no longer needed.

File: Triaging-process.md (hunk 21 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

File: Triaging-process.md (hunk 22 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

File: Triaging-process.md (hunk 20 above)
Comment (Lawful2002) [reply Triaging-process.md]: Done

**Comment by Lawful2002** [comment]
@gp201 PTAL.

**Comment by Lawful2002** [comment]
@gp201 Can you please take a look?

File: Triaging-process.md [hunk 24]
```diff
@@ -0,0 +1,62 @@
...
+- If you’re not sure how to categorise an issue, feel free to ping Sean (@seanlip) for help!
+
+### Second-stage triage - Steps to triage incoming issues for each team:
+Team leads should do an audit every week to categorise new issues in their GitHub Projects board, following these steps:
+
+1. Verify that the issue lies within the scope of the team (see definitions below).
+    - If not, assign the issue to the appropriate project/team.
+2. Determine whether the issue is a bug or a enhancement
+    - If the issue is a feature request (enhancement), contact the relevant PM (product manager) for the team and ask them to take a look and decide whether it needs to be worked upon. (See the list of PMs [below](#pm-contacts-for-each-team).)
+3. (For simple UI issues) Make sure the bug is reproducible
```
Comment (gp201) [inline Triaging-process.md:19]: In case it is not a UI issue then what should the devs do?

Comment (gp201) [reply Triaging-process.md:19]: Try and mention the step the devs should take and list the exception or points to be noted as subpoints.

**Review by gp201 — CHANGES_REQUESTED** [review]
@Lawful2002 1 final comment.

File: Triaging-process.md (hunk 24 above)
Comment (Lawful2002) [reply Triaging-process.md:19]: Done, PTAL.

File: Triaging-process.md [hunk 25]
```diff
@@ -0,0 +1,64 @@
...
+1. Verify that the issue lies within the scope of the team (see definitions below).
+    - If not, assign the issue to the appropriate project/team.
+2. Determine whether the issue is a bug or a enhancement
+    - If the issue is a feature request (enhancement), contact the relevant PM (product manager) for the team and ask them to take a look and decide whether it needs to be worked upon. (See the list of PMs [below](#pm-contacts-for-each-team).)
+    - If the issue is a bug, add the **Bug** label.
+3. (For simple UI issues) Make sure the bug is reproducible
+    - The issue should be consistently reproducible on either develop or the test/backup server.
+    - If that isn’t the case, ask the issue author for clarity.
+    - If the issue is not a simple UI bug that can be easily verified, the task of verifying the reproducibility lies on the developer working on the issue.
+4. Once the issue has been confirmed, the team lead should add the  appropriate ‘Impact’ label (asking the PM or tech lead for help if they’re not sure), as well as the appropriate ‘Work’ label to the issue.
```
Comment (gp201) [inline Triaging-process.md]: What are the criteria for an issue getting confirmed? Do you mean "Once the issue has been created"?

**Review by gp201 — CHANGES_REQUESTED** [review]
last comment

File: Triaging-process.md (hunk 25 above)
Comment (Lawful2002) [reply Triaging-process.md]: I have made some changes. PTAL.

**Review by gp201 — APPROVED** [review]
LGTM

**Pull request merged**